packages/s3/lib/streams.js

```diff
@@ -34,3 +34,63 @@ export const createWritableStream = (path) => {
   const stream = createWriteStream(path)
   return toWritableStream(stream)
 }
+
+/**
+ * Convert a ReadableStream to a Readable.
+ *
+ * @param {any} webStream The WebStream to convert.
+ * @returns {Readable} The Node.js stream.
+ */
+export const toReadable = (webStream) => {
+  const nodeStream = new Readable({
+    // The read method is called when the stream wants to pull more data in
+    read: () => { },
+  })
+
+  // Handle each chunk of data
+  const reader = webStream.getReader()
+  const pump = () => {
+    reader.read().then(({ done, value }) => {
+      if (done) {
+        // When no more data, close the stream
+        nodeStream.push(null)
+      } else {
+        // Push the data into the Node.js stream
+        nodeStream.push(Buffer.from(value))
+        pump()
+      }
+    }).catch(err => {
+      // Handle any errors
+      nodeStream.emit('error', err)
+    })
+  }
+
+  // Start pumping data
+  pump()
+
+  return nodeStream
+}
```

Comment:
Would is be possible to use `Readable.fromWeb`?
Similarly, could `toWritableStream` be replaced with `Writable.toWeb`?

Reply:
> Would is be possible to use `Readable.fromWeb`?

It doesn't seem to work.
If you find another solution, we can try, else I suggest keeping this for now.

> Similarly, could `toWritableStream` be replaced with `Writable.toWeb`?

For this one, I was able to replace it.

Reply:
> It doesn't seem to work.

Do you remember what failed? I simply replaced `toReadable` with `Readable.fromWeb` and tests passed

Reply:
I had a test failing (the one for the `toReadable` function ; if I replaced the call with `Readable.fromWeb` it was giving the following error: `TypeError [ERR_INVALID_ARG_TYPE]: The "readableStream" argument must be an instance of ReadableStream. Received an instance of Object`).
So I just removed the test ; the rest is passing.

I also needed to add a `// @ts-ignore` comment so that it doesn't complain anymore with types issues.